apps/nestjs-indexer/src/services/token-launch/token-launch.service.ts

```diff
@@ -33,9 +40,9 @@ export class TokenLaunchService {
           price: data.price,
           total_supply: data.totalSupply,
           current_supply: data.totalSupply,
-          is_liquidity_added: false,
+          is_liquidity_added: true,
```

Comment:
The liquidity is not added so by default it's false here because just launched.. 

We have another indexer for the event LiquidityAdded: change to LiquidityCreated

In this one we need to change the liquidity to true